test/unit-tests/testIq.js

```diff
@@ -412,7 +412,8 @@ describe('Analytics Framework Template Unit Tests', function()
     var unitTestState = plugin.ooyalaReporter.unitTestState;
     // Temporarily disable this check
     //expect(unitTestState.reportPlayRequestedCalled).toBe(1);
-    expect(plugin.getAutoPlay()).toBe(false);
+    // Temporarily disable this check until we resolve autoplay
+    //expect(plugin.getAutoPlay()).toBe(false);
```

Comment:
I think autoplay is supposed to be all lower case. That may be why this wasn't working.

Reply:
I commented out setting autoplay in the plugin per @aeng7's comments - basically we weren't sure if this represented a "true" autoplay event and it looked like it was just set from the player page level params. It causes this check to fail since the autoplay variable never receives the value in the metadata blob

Reply:
Now I remember. Sounds good to me then.